src/IB/IBStandardForceGen.cpp

```diff
@@ -1267,6 +1267,7 @@ IBStandardForceGen::computeLagrangianTargetPointForce(Pointer<LData> F_data,
     double max_displacement = 0.0;
 
     const int num_target_points = static_cast<int>(d_target_point_data[level_number].petsc_node_idxs.size());
+    if (num_target_points == 0) return;
```

Comment:
I would be careful here --- I think there are some collective operations that happen in this routine, and so the routine really should be rewritten to work when `num_target_points` is `0`.

Reply:
I took a look and it seems like this routine (before my changes) doesn't do anything if `num_target_points` is `0`. Both the loops will exit without any iterations. Could you confirm when you get a chance; maybe I'm missing something?